plugin/ale.vim

```diff
@@ -32,13 +32,22 @@ if !s:has_features
     finish
 endif
 
-" This flag can be set to 0 to disable emitting conflict warnings.
-let g:ale_emit_conflict_warnings = get(g:, 'ale_emit_conflict_warnings', 1)
-
-if g:ale_emit_conflict_warnings
-    " Add the after directory to the runtimepath
+" Add the after directory to the runtimepath
+let s:sep = has('win32') ? '\' : '/'
+let s:should_add_after_dir = 1
+for s:path in split(&runtimepath, ',')
+    if s:path =~# s:sep . 'after$' &&
+    \  filereadable(s:path . s:sep . 'plugin' . s:sep . 'ale.vim')
+        let s:should_add_after_dir = 0
+        break
+    endif
+endfor
+unlet! s:sep
+unlet! s:path
+if s:should_add_after_dir
     let &runtimepath .= ',' . expand('<sfile>:p:h:h') . '/after'
 endif
+unlet! s:should_add_after_dir
```

Comment:
This is too complicated. Just search for something like `'ale[/\\]after'` in `runtimepath` instead.

Reply:
`match('ale[/\\]after$', &runtimepath) >= 0` should be enough.

Reply:
```vim
if g:ale_emit_conflict_warnings && match('ale[/\\]after$', &runtimepath) < 0
```